src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -243,6 +243,27 @@
      * modifiers, type, the identifier name, and an optional
      * assignment statement.
      *
+     * <p>For example:</p>
+     *
```

Comment:
lets stick to similar style like [here](https://github.com/checkstyle/checkstyle/pull/9378/files), without empty comment lines

Reply:
Done